File: cases/taint/python/array-element-negative/array_element.py
def dfb_source():  # DFB-SOURCE: array-element-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: array-element-sink
    pass


def run():
    values = [None, None]
    values[0] = dfb_source()  # DFB-WITNESS: array-element-store
    values[1] = "clean"
    dfb_sink(values[1])

File: cases/taint/ruby/array-element-positive/array_element.rb
def dfb_source # DFB-SOURCE: array-element-input
  "tainted"
end

def dfb_sink(value) # DFB-SINK: array-element-sink
end

def run
  values = [nil, nil]
  values[0] = dfb_source # DFB-WITNESS: array-element-store
  values[1] = "clean"
  dfb_sink(values[0])
end
